### DISCUSSION TITLE
sudo

### DISCUSSION THREAD

**Discussion by gngrossi** [zopencommunity/meta#224] [Reference]
I may have missed it, is there an upcoming plan for a sudo port?
thanks

**Comment by IgorTodorovskiIBM** [comment]
It's on our list. In fact, I started working on it a bit last week. 
I placed my work in progress changes under https://github.com/ZOSOpenTools/sudoport. The current issues are that `pw_passwd` is not a supported member in the z/OS `struct passwd`. Also the install step requires root to run successfully.

**Reply by gngrossi** [reply]
Good news.
We are running sudo 1.7.2 (IBM Ported Tools distribution) which is no longer available/supported. sudo without Kerberos support would be a good start. thanks

**Comment by IgorTodorovskiIBM** [comment]
Hi Gary, sudo is now available under https://github.com/ZOSOpenTools/sudoport/releases/tag/Releaseline_sudoport_1014.

**Comment by MikeFultonDev** [comment]
@gngrossi did you have a chance to try out sudo? If so, any feedback?

**Comment by gngrossi** [comment]
https://github.com/ZOSOpenTools/sudoport/issues/4

**Comment by MikeFultonDev** [comment]
Perfect. Thanks for opening the issue(s). Can I close this off and we can tackle this via issues?

**Comment by gngrossi** [comment]
Yes, please do...thanks
